### ISSUE TITLE
`pygetpapers` without options causes traceback

### ISSUE THREAD

**Issue by petermr** [petermr/pygetpapers#6]
Issuing`pygetpapers` gives the following:
```
pygetpapers
INFO: */submitting RESTful query (I)*/
INFO: */RESTful request for fulltext.xml (D)*/
INFO: */Got the Query Result */
INFO: Time elapsed: 0.16755104064941406
Traceback (most recent call last):
  File "/opt/anaconda3/bin/pygetpapers", line 8, in <module>
    sys.exit(main())
  File "/opt/anaconda3/lib/python3.8/site-packages/pygetpapers/pygetpapers.py", line 504, in main
    callpygetpapers.handlecli()
  File "/opt/anaconda3/lib/python3.8/site-packages/pygetpapers/pygetpapers.py", line 491, in handlecli
    self.apipaperdownload(args.query, args.limit,
  File "/opt/anaconda3/lib/python3.8/site-packages/pygetpapers/pygetpapers.py", line 370, in apipaperdownload
    query_result = self.europepmc(query, size)
  File "/opt/anaconda3/lib/python3.8/site-packages/pygetpapers/pygetpapers.py", line 128, in europepmc
    if "nextCursorMark" in builtquery["responseWrapper"]:
KeyError: 'responseWrapper'
```

**Comment by petermr** [comment]
Recommendation:
This should call 
```
pygetpapers --help
```
and should NOT attempt to submit a query

**Comment by ayush4921** [comment]
Fixed in version 1.0.2
